testing/configs/config.yml

```diff
@@ -1,4 +1,4 @@
-custom_transform: customtransforms/custom_test1.py
+custom_transform: perftestnotebook/customtransforms/custom_test1.py
```

Comment:
This will be fine for now, but we will need to change how we define the Transformer. See this issue: https://github.com/mozilla/mach-perftest-notebook-dev/issues/25